sdks/python/apache_beam/io/external/xlang_bigqueryio_it_test.py

```diff
@@ -260,33 +262,43 @@ def run_streaming(
         streaming=True,
         allow_unsafe_triggers=True)
 
+    auto_sharding = num_streams == 0
```

Comment:
Took me some time to parse it a bit, nit and might be common practice, but `auto_sharding = (num_streams == 0)` looks better